tutorials/performance/gpu_optimization.rst

```diff
@@ -175,7 +175,8 @@ amount of work the GPU has to do. You can do this by simplifying the shader
 <class_StandardMaterial3D>`), or reducing the number and size of textures used.
 
 **When targeting mobile devices, consider using the simplest possible shaders
-you can reasonably afford to use.**
+you can reasonably afford to use.** Also, when using non-unshaded particles,
```

Comment:
"Non-unshaded" is a double negative, but since the material flag is called "Unshaded", I'd prefer explicitly mentioning it.

When using an unshaded material, the shading mode doesn't matter.

Reply:
Also I'm wondering if moving the mention of vertex shading might be better moved up to join with the paragraph above, I think it might flow better.